README.md

```
@@ -1,13 +1,44 @@
+- Keycloak client with:
+    - `GLOBAL_ADMIN`
+    - `PARTNER_ADMIN`
+    - `ID_AUTHENTICATION`
+-  Docker installed.
 * If the `ENABLE_INSECURE` environment variable is set to `true`, the script will proceed with downloading an SSL certificate and subsequently provide it for utilization in **Newman** collections and **curl** API calls during execution. This functionality is designed for scenarios where the script is required to be used on a server that possesses self-signed SSL certificates.
+
+## Contribution & Community:
+
+- We welcome contributions from everyone!
+
+- Check [here](https://docs.mosip.io/1.2.0/community/code-contributions) to learn how you can contribute code to this application.
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)